runtime/_data.ts

```diff
@@ -292,7 +292,7 @@ export async function generateDescriptions(): Promise<Descriptions> {
     )
   ) {
     const file = await Deno.readTextFile(dirEntry.path);
-    const parsed = yamlParse(file);
+    const parsed = yamlParse(file) as DescriptionItem;
```

Comment:
The code after here is for validating that `parsed` is in the form of `DescriptionItem`. It doesn't seem correct to cast this type to `DescriptionItem` here